src/components/berths/index.tsx

```diff
@@ -1,49 +1,27 @@
 import React from 'react';
 import { FormattedMessage } from 'react-intl';
 import { Col, Container, Row } from 'reactstrap';
-import styled from 'styled-components';
 
 import Berth from './Berth';
 
-import { Berths as BerthsType, SelectedBerths } from '../../types/berths';
-
-interface Props {
-  filtered: BerthsType;
-  filteredNot: BerthsType;
-  selected: SelectedBerths;
-  onClick: Function;
-  TabHeader?: React.FC;
-}
-
-const StyledBerth = styled(Berth)`
-  opacity: ${props => (props.excluded ? '0.5' : '1')};
-`;
-
-const Wrapper = styled(Container)`
-  ${StyledBerth} + ${StyledBerth} {
-    margin-top: 1em;
-  }
-`;
-
-const ListHeader = styled.h3`
-  margin: 1em 0;
-`;
+import { BerthProps } from './types';
 
 const { REACT_APP_MAX_SELECTED_BERTHS } = process.env;
 
-export default ({ filtered, filteredNot, onClick, selected }: Props) => (
-  <Wrapper>
+export default ({ filtered, filteredNot, onClick, selected }: BerthProps) => (
+  <Container className="app-berths">
```

Comment:
Component names are sometimes capitalized and sometimes not, I prefer to stick to one or the other.

Reply:
capital it is, will fix